### PR TITLE
Antd v4 including update analysis pages.

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Full update to Ant Design v4

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/linelist/LineListController.java [hunk 1]
```diff
@@ -36,18 +36,20 @@
...
 @Controller
 @RequestMapping("/linelist")
 public class LineListController {
-	private ProjectService projectService;
-	private SampleService sampleService;
-	private MetadataTemplateService metadataTemplateService;
-	private MessageSource messages;
-	private UpdateSamplePermission updateSamplePermission;
-	private ProjectOwnerPermission projectOwnerPermission;
+	private final ProjectService projectService;
```
Comment (deepsidhu85) [inline src/main/java/ca/corefacility/bioinformatics/irida/ria/web/linelist/LineListController.java]: The `final` variable should be removed here as well no?

Comment (joshsadam) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/linelist/LineListController.java]: Think I got it all fixed in : c4f7320ebf7b82fad9636119048b65bec9f20718

File: src/main/webapp/resources/js/pages/analysis/components/settings/AnalysisReferenceFileRenderer.jsx [hunk 2]
```diff
@@ -29,30 +30,28 @@ export function AnalysisReferenceFileRenderer() {
...
         <div style={{ marginBottom: SPACE_LG }} key="samplesDiv-1">
           <Title level={4}>{i18n("AnalysisSamples.referenceFile")}</Title>
-          <Row key="row-reference-file-1">
-            <span
-              key="reference-file-1"
-              style={{
-                display: "flex",
-                justifyContent: "space-between",
-                alignItems: "center"
+          <dive
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/analysis/components/settings/AnalysisReferenceFileRenderer.jsx]: Incorrect div open and close tag?

Comment (joshsadam) [reply src/main/webapp/resources/js/pages/analysis/components/settings/AnalysisReferenceFileRenderer.jsx]: Fixed

**Pull request merged**